yaksh/templates/exam.html

```diff
@@ -26,6 +26,13 @@
     </nav><!-- /.navbar -->
 {% endblock %}
 {% block content %}
+  <script>
+    $(document).ready(function() {
+        var ele = document.getElementById('error_panel')
+        ele.scrollIntoView(true);
```

Comment:
This will give you JS errors since ele will be undefined, please write this better.